pkg/util/utils.go

```diff
@@ -34,11 +34,13 @@ import (
 	"k8s.io/minikube/pkg/minikube/out"
 )
 
-// ErrPrefix notes an error
-const ErrPrefix = "! "
+const (
```

Comment:
Thanks for the PR. Do you mind also adding the `downloadURL` on the `const` block below to the same block here. pls?

Reply:
@josedonizetti Thank you for your comments! 
Yes, I considered putting `downloadURL` into a `constant block` before I created this pr，but I don't think the two `constant blocks` define the same thing. all I didn't define together.
Are you sure? I can update pr

Reply:
yes I agree with @josedonizetti it looks nicer to have all of them in same block.